### ISSUE TITLE
Please allow a toggle to include the "premium 1080P bitrate" option in video quality settings

### ISSUE THREAD

**Issue by AndrewGrill1147** [dessant/youtube-video-quality#26]
**Is your feature request related to a problem? Please describe.**
When selecting 4K as my default, the add-on properly defaults to 1080P if that's the highest available, but does not select the premium 1080P bitrate option where available.

**Describe the solution you'd like**
Please add either a toggle button to enable 1080P premium when either it or a higher resolution are selected, or add 1080P premium to the default list above regular 1080 when available.

This addon is really great and seems to work near-perfectly!  This would be a nice cherry on top :)

**Comment by Andreyf20** [comment]
Yeah same problem here. I think this would be perfect if it chose the 1080 premium when its available as the highest "resolution", or maybe a toggle similar to the limit to 30 fps option.

**Comment by MuhannadYT** [comment]
another issue is when selecting 1080p premium bitrate is changes back to the lower bitrate option

**Comment by rfischmann** [comment]
Another vote for this!
